javascript/operators/spread.json

```
@@ -5,6 +5,11 @@
         "__compat": {
           "description": "Spread syntax (...)",
           "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Spread_syntax",
+          "spec_url": [
+            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-SpreadElement",
+            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-ArgumentList",
+            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-PropertyDefinition",
```

Small syntax error; JSON doesn't like tailing commas!

```suggestion
            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-PropertyDefinition"
```

---

Oh oops, thanks for catching